redbot/cogs/trivia/data/lists/minecraft.yml

```
@@ -0,0 +1,114 @@
+AUTHOR: ImajinDevon
+
+"The phantom was added in which update?":
+ - "1.13"
+"Which update introduced a new style combat into the game?":
+ - "1.9"
+'What edition of Minecraft is usually prefixed with the word "bug"?':
+ - "Bedrock"
+"Who created Minecraft?":
+ - "Notch"
+"Who bought Mojang in 2014?":
+ - "Microsoft"
+"Mojang was bought for (...) billion dollars:":
+ - "2.5"
+"If you name a sheep this, it will turn rainbow:":
+ - "jeb_"
+"What supercharges creepers?":
+ - "lightning"
+"What are creepers scared of?":
+ - "cats"
+"To kill an iron golem without getting hit, you must build how many blocks up?":
+ - "3"
+"What is an enderman's weakness?":
+ - "water"
```

```suggestion
"What is an enderman's weakness?":
 - "water"
 - "Water"
 - "fire"
 - "Fire"
 - "lava"
 - "Lava"
 - "rain"
 - "Rain"
```

---

I think this is more of an issue of how the question is worded. Endermen are _semi uniquely_ damaged by water:
> Water damages endermen, snow golems, striders and blazes, at a rate of 1♥ per half second.
> Endermen can be harmed by melee attacks, water, lava, fire, splash water bottles, or rain.
```suggestion
"What is one mob that is damaged by water?":
 - "endermen"
 - "enderman"
 - "snow golem"
 - "strider"
 - "blaze"
```